packages/rollup-full-node/README.md

```diff
@@ -9,6 +9,23 @@ Run `yarn install` to install necessary dependencies.
 # Building
 Run `yarn build` to build the code. Note: `yarn all` may be used to build and run tests.
 
+## Building Docker Image
+Install [docker]()
```

Comment:
Should technically be an extra newline between this and the next line otherwise it will all show up on the same line. -- https://github.com/ethereum-optimism/optimism-monorepo/blob/ee10d4186dfc7511f2d1aba2df49dc2f97a6ecca/packages/rollup-full-node/README.md